core-web/libs/block-editor/src/lib/extensions/bubble-form/plugins/bubble-form.plugin.ts

```diff
@@ -143,7 +143,7 @@ export class BubbleFormView extends BubbleMenuView {
 
         this.tippy = tippy(editorElement.parentElement, {
             ...this.tippyOptions,
-            duration: 250,
+            duration: [250, 0],
```

Comment:
Rafa, these values can be a constant? `const DURATION = 250`.

Reply:
Sure!